### ISSUE TITLE
Working directory

### ISSUE THREAD

**Issue by Tommuh** [nmaier/mod_xsendfile#20]
First of all I would like to thank you for this nice piece of software.

I have a question, it is not really an issue. I do not know where to post else my question.

Is there a way to disable the working directory?
I want Apache only look in the directories I have white listed and not where I execute it.

Big thanks in advance!

**Comment by nmaier** [comment]
Nope, (hence I leave this open), but for now you can simply patch out the corresponding code.
